services/market/shared/market/index.js

```diff
@@ -0,0 +1,29 @@
+/*
+ * LiskHQ/lisk-service
+ * Copyright © 2021 Lisk Foundation
+ *
+ * See the LICENSE file at the top-level directory of this distribution
+ * for licensing information.
+ *
+ * Unless otherwise agreed in a custom licensing agreement with the Lisk Foundation,
+ * no part of this software, including this file, may be copied, modified,
+ * propagated, or distributed except according to the terms contained in the
+ * LICENSE file.
+ *
+ * Removal or modification of this copyright notice is prohibited.
+ *
+ */
+const {
+    getMarketPrices,
+} = require('./market');
+
+const {
+    reloadPricesFromBinance,
+    getBinancePricesFromDB,
+} = require('./sources');
+
+module.exports = {
+    getMarketPrices,
+    reloadPricesFromBinance,
+    getBinancePricesFromDB,
+};
```

Comment:
We probably do not need to export the source-specific methods via the market API since we don't invoke them anywhere else apart from the jobs.

```suggestion
module.exports = {
    getMarketPrices,
};
```